docs/TROUBLE.md

```diff
@@ -0,0 +1,175 @@
+
+- If you use "Rx Snd RcChannel" = "rc overwrite" setting instead of connecting a CRSF receiver output and are testing with QGroundControl, switch to Mission Planner or APM Planner 2.0 since QGroundControl doesn't show data from RC\_CHANNELS\_OVERRIDE MAVLink messages on this page.
+
+- If the display doesn't follow stick movement and you use a CRSF connection between your receiver and your flight controller, check that you have connected the wire to the correct pins/pads on both your receiver and flight controller.
+
+- For CRSF, check that you have correctly [configured ArduPilot](ARDUPILOT.md#crsf-receiver) to use this input.
+
+- For CRSF, check that you have correctly [configured your Rx](CRSF.md#mlrs-rx-module-setup) for crsf output with "Rx Out Mode" = "crsf".
+
+- If your receiver [doesn't provide an RC output](ELRS_RECEIVERS.md#connections) or you don't want to use it, check that you have set ["Rx Snd RcChannel"](PARAMETERS.md#rx-snd-rcchannel) to "rc override".
+
+## 9. Confirm reliable message delivery
+
+Use your ground control station connected to your Tx module to examine the message loss rate both during parameter download and for a few minutes after it completes.
+
+For QGroundControl, select "Application Settings" -> "MAVLink" and scroll down to "MAVLink Link Status".
+
+For Mission Planner, select "status" in the upper right, move the windows so you can see both and then refresh parameters from the "full parameters" page.
+
+You should see very few or no messages lost.
+
+- If you see more than the occasional lost message or if parameter download takes more time than expected (8 to 60 seconds, depending on mode and ArduPilot version), check that the [Rx Ser Link Mode"](PARAMETERS.md#rx-ser-link-mode) parameter is set to "mavlink" or "mavlinkX" and that ["Rx Snd Radiostat"](PARAMETERS.md#rx-snd-radiostat) is set to "ardu_1" or "meth_b"
+
+- Depending on your ArduPilot version, it may help to adjust your [ArduPilot setup](CRSF.md#ardupilot-setup) to use the recommended baud rate between the receiver and flight controller;  Also adjust your configured stream rates as appropriate for your radio mode.
+
+- Upgrade to a newer version of ArduPilot which may have fixed flow control issues.
+
+## Range issues
+
+If you don't see the range you expected, consider the following.
+
+- Powering up a receiver or Tx module without an antenna connected may cause damage, especially at high output levels.
+
+- When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is unfortunately possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal (see previous bullet).
+
+- If your vehicle allows, always use a vertical antenna orientation on both the Tx module and the Rx.  Especially with non-diversity systems.
```

Comment:
Nitpick - guess I would prefer 'single antenna systems.'

Reply:
Nitpick right back; then they might count their video antenna.  I know it's clumsy, but I prefer non-diversity here.

Reply:
Fair enough